lib/scripts/pdf_a4_portrait.coffee

```diff
@@ -21,6 +21,7 @@ exit('Did not receive any html') if !json.html?.trim()
 options = json.options
 page = webpage.create()
 page.content = json.html
+nbPages = 0
```

Comment:
I changed it.
The only problem with this, is if you did not have an header or footer, you will not have the right number of pages :/